library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -492,6 +492,26 @@ bool libspdm_generate_cert_chain_hash(libspdm_context_t *spdm_context,
         hash);
 }
 
+/**
+ * Get the certificate slot mask
+ *
+ * @param[in]   context              A pointer to the SPDM context.
+ * @param[out]  slot_mask            address of get slot mask
+ *
+ **/
+void libspdm_get_cert_slot_mask_and_count(libspdm_context_t *spdm_context,
+                                          uint8_t *slot_mask)
```

Comment:
I think it should be:

```
uint8_t libspdm_get_cert_slot_mask (libspdm_context_t *spdm_context);
```

Reply:
Thanks. I forgot change the function name before. I have changed it.